.github/workflows/runner.yml

```diff
@@ -23,6 +23,17 @@ jobs:
         run: python -m mypy --config-file mypy.ini -p ethstaker_deposit
       - name: Run linter
         run: flake8 --config=flake8.ini ./ethstaker_deposit ./tests
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+      - name: Install jsonlint
+        run: npm install -g @prantlf/jsonlint
+      - name: Validate JSON files
+        run: |
+          for file in $(find ./ethstaker_deposit/intl -name "*.json"); do
+            jsonlint $file || exit 1
```

Comment:
Could we add the `--quiet` param so we don't output the parsed json files.  Seems a little _loud_: https://github.com/yorickdowne/ethstaker-deposit-cli/actions/runs/10650339689/job/29521585977#step:9:14

Reply:
Good point :sweat_smile: . Done.